### ISSUE TITLE
Mobile client

### ISSUE THREAD

**Issue by mrbluecoat** [husarnet/husarnet#60]
Cool project! Any ETA update on Android support?

Also, https://github.com/husarnet/husarnet/blob/master/README.md?plain=1#L12 says Windows support is coming soon but your codebase and releases page looks like it's already supported?

Lastly, any plans for iOS?

ref #17

**Comment by DominikN** [comment]
Thanks @mrbluecoat for your feedback!

Here's a status of Windows/Android/iOS clients:

1. Windows - public beta version available [here](https://husarnet.com/docs#using-husarnet) (under **Windows** tab)
2. Android - experimental, internal version. Will be released after 2.0.0 version of Husarnet Client (autumn 2022).
3. iOS - Will be released after 2.0.0 version of Husarnet client and after Android version.

Basically 2.0.0 version of Husarnet Client will share much more code base for different platforms than the current one. Thanks to that porting and maintanace of versions for different OSes will be faster and simpler with no additional technical debt.

**Comment by mrbluecoat** [comment]
With v2 released, any update on the mobile clients?

**Comment by pidpawel** [comment]
Unfortunately not a whole lot is going on in the mobile client space. I hope we will be able to start working on it before the end of this year though.

**Comment by mrbluecoat** [comment]
No worries and thanks for a nice service

**Comment by kscheff** [comment]
Any update on the mobile client front? I am in search for a P2P communication from an ESP32C3 device to a smartphone App. The solution needs to be deployable to both Android and iOS.

**Comment by fwlodarczyk** [comment]
> Any update on the mobile client front? I am in search for a P2P communication from an ESP32C3 device to a smartphone App. The solution needs to be deployable to both Android and iOS.

We are working on Husarnet client for Android , Apple iOS version isn't on our roadmap unfortunately.

**Comment by fwlodarczyk** [comment]
> Missing out on iOS seems to me not a smart move, since in several key markets the share is in favor of iOS. Especially looking into revenue, high disposable income iOS dominates. Please read: https://explodingtopics.com/blog/iphone-android-users#
> 
> "IPhone Vs Android User Stats (2023 Top Picks)
> 
> Here’s a quick look at the difference between iPhone and Android users:
> 
> In the US, iPhone has a 57.93% market share. As of 2023, Android has a 70.29% global market share. iPhone users have a 43.7% higher average salary than Android users. Gen Z and Millennials are more likely to be iPhone users. 47% of former Android users say iPhone provides a better user experience. 29% of former iPhone users say Android is more affordable."

We're aware of the statistics, and it's definitely on our radar - we simply don't have any estimates when we'll start working on it.

**Comment by guiedup** [comment]
Hello, any update on the mobile clients?

**Comment by ympek** [comment]
Hello
We a couple of weeks close to releasing iOS client to alpha, and we will release it alongside with the new stable daemon.
Android - still no idea honestly, maybe next year Q1 or Q2

Regards
